src/sentry/static/sentry/app/views/settings/organizationMembers/inviteRequestRow.tsx

```diff
@@ -53,87 +55,95 @@ const InviteRequestRow = ({
         <h5 style={{marginBottom: '3px'}}>
           <UserName>{inviteRequest.email}</UserName>
         </h5>
-        {inviteRequest.inviteStatus === 'requested_to_be_invited' ? (
-          inviteRequest.inviterName && (
-            <Description>
-              <Tooltip
-                title={t(
-                  'An existing member has asked to invite this user to your organization'
-                )}
-              >
-                {tct('Requested by [inviterName]', {
-                  inviterName: inviteRequest.inviterName,
-                })}
-              </Tooltip>
-            </Description>
-          )
-        ) : (
-          <Tooltip title={t('This user has asked to join your organization.')}>
-            <JoinRequestIndicator size="small">{t('Join request')}</JoinRequestIndicator>
-          </Tooltip>
-        )}
+        {hasWriteAccess &&
+          (inviteRequest.inviteStatus === 'requested_to_be_invited' ? (
```

Comment:
Weird that `requested_to_be_invited` is not a constant but this probably goes way back.

It may be a good idea to follow up in a separate PR to refactor all uses to a constant just to clean things up.